atlasdb-cassandra/src/main/java/com/palantir/atlasdb/cassandra/CassandraKeyValueServiceConfig.java

```diff
@@ -43,6 +43,11 @@ public int poolSize() {
         return 20;
     }
 
+    @Value.Default
+    public int poolRefreshIntervalSeconds() {
+        return 5 * 60;
```

Comment:
I don't have strong feelings about dropping down from 10 to 5 minute default, but we should document it... Somewhere